modules/tinymce/changelog.txt

```diff
@@ -22,6 +22,7 @@ Version 5.4.0 (TBD)
     Fixed an issue where removing formatting in a table cell would cause Internet Explorer 11 to scroll to the end of the table #TINY-6049
     Fixed an issue where the `allow_html_data_urls` setting was not correctly applied #TINY-5951
     Fixed the `autolink` feature so that it no longer treats a string with multiple "@" characters as an email address #TINY-4773
+		Changed the `link` plugin to suggest `mailto:` based on the absence of URL separators (/'s) #TINY-5941
```

Comment:
Indentation issue here, as the line is offset and additionally changed should be before fixed lines.

In saying that though, this shouldn't be a `Fixed` line as remember the changelog is from a users point of view. So there's nothing changing here from their perspective, it's just fixing something that shouldn't have been picked up as a possible email.

- [ ] Change this to `Fixed` and reword (plus fix indentation)